ush/config_defaults.yaml

```diff
@@ -330,11 +411,14 @@ platform:
   # System directory where the lookup tables for optics properties are located
   #
   # FIXorg:
-  # System directory where orography data is located
+  # System directory where orography data are located
```

Comment:
Ooh, is this where we debate the merits of "data is" vs. "data are"? 😈

Reply:
Now I'm curious about your opinion/reasoning... altho maybe our CMs would prefer us to take that discussion elsewhere 😄 Not gonna lie, there have been times where I either completely overlook this error (as in every other time I've read this chapter) or purposely add an error because "data are" sounds too weird in context.

Reply:
I actually have no interest in legislating this one, I just know it's a potentially controversial one haha